index.html

```diff
@@ -0,0 +1,390 @@
+            <p class="header__content__subtitle">Very suitable to support all web development project</p>
+            <div class="header__content__buttons">
+                <a class="header__content__button">OUR SERVICES</a>
+                <a class="header__content__button">HERE IS NOW</a>
+            </div>
+        </div>
+    </header>
+    <main>
+        <div class="about-us">
+            <div class="container">
+                <div class="main__name">
+                    <p class="main__name__title">About Us</p>
+                    <p class="main__name__subtitle">Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti quos porro,<br> ipsa sed 
+                        distinctio consectetur voluptate magnam esse atque quia cumque sequi</p>
+                </div>
+                <div class="about-us__info">
+                    <div class="about-us__content">
+                        <img class="about-us__content__img" src="img/globe.png"/>
+                        <p class="about-us__content__title">Awesome Icons</p>
+                        <p class="about-us__content__subtitle">Lorem ipsum dolor sit amet consectetur adipisicing elit, sed do eiusmod tempor</p>
+                    </div>
+                    <div class="about-us__content">
+                        <img class="about-us__content__img" src="img/transfer.png"/>
+                        <p class="about-us__content__title">One Page</p>
+                        <p class="about-us__content__subtitle">Lorem ipsum dolor sit amet consectetur adipisicing elit, sed do eiusmod tempor</p>
+                    </div>
+                    <div class="about-us__content">
+                        <img class="about-us__content__img" src="img/head.png"/>
+                        <p class="about-us__content__title">Fully Responsive</p>
+                        <p class="about-us__content__subtitle">Lorem ipsum dolor sit amet consectetur adipisicing elit, sed do eiusmod tempor</p>
+                    </div>                    
+                </div>
+            </div>
+        </div>
+        <div class="our-services">
+            <div class="container">
+                <div class="main__name">
+                    <p class="main__name__title">Our Services</p>
+                    <p class="main__name__subtitle">Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti quos porro,<br> ipsa sed 
+                        distinctio consectetur voluptate magnam esse atque quia cumque sequi</p>
+                </div>
+                <div class="our-services__info">
+                    <div class="our-services__content">
+                        <div class="our-services__content__img"></div>
+                        <p class="our-services__content__title">Research</p>
+                        <p class="our-services__content__subtitle">At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis 
```

Comment:
Add paddings 20px left/right to this block

Reply:
done